### PR TITLE
added homepage in jsx

### PR DESCRIPTION
**Pull request by Aman-Jain1996**
added homepage in jsx

### CODE REVIEW

File: src/components/NavBar/Navbar.css [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+header {
+  display: flex;
+  align-items: center;
+  background-color: var(--bg-lightWhite);
+  height: 12vh;
+  padding: 2.4rem 4.8rem;
+  justify-content: space-between;
+  position: sticky;
+  top: 0;
+  /* box-shadow: 0 4px 12px 0 rgba(0 , 0 ,0 , .8); */
```
Comment (romabulani) [inline src/components/NavBar/Navbar.css:12]: You can remove this comment if not required

File: src/pages/Home/Home.css [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+  animation: text-overlay 1s ease-in-out 0.2s;
+}
+
+@keyframes text-overlay {
+  0% {
+    top: 150%;
+  }
+  100% {
+    top: 50%;
+    /* transform:translate(-50%,-50%); */
```
Comment (romabulani) [inline src/pages/Home/Home.css:36]: Here also you can remove the comment

**Review by romabulani — APPROVED** [review]
- Clean code
- Easy to read
- Just a suggestion, you can host the images on cloudinary, it will help in fast loading

File: src/index.css [hunk 3]
```diff
@@ -1,13 +1,91 @@
-body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+@import url('https://fonts.googleapis.com/css2?family=Ubuntu:ital,wght@1,700&display=swap');
```
Comment (Shah-Arsalan) [inline src/index.css:1]: You can import this in app.js as well

File: src/pages/Home/Home.jsx [hunk 4]
```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import Navbar from "../../components/NavBar/Navbar";
+import "./Home.css";
+import men from "../../assets/images/men.webp";
```
Comment (Shah-Arsalan) [inline src/pages/Home/Home.jsx:4]: Recently came to know about barrel exports . I think that would be a better practice

**Review by Shah-Arsalan — APPROVED** [review]

File: src/pages/Home/Home.jsx (hunk 4 above)
Comment (Aman-Jain1996) [reply src/pages/Home/Home.jsx:4]: I will do that once completed all functionalities

File: src/pages/Home/Home.css (hunk 2 above)
Comment (Aman-Jain1996) [reply src/pages/Home/Home.css:36]: Sure Roma ,  will do that

**Pull request merged**